cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -235,6 +235,11 @@ protected ClientConfig getClientConfig() {
       builder.setAccessToken(accessToken);
     }
 
+    String verifySSL = System.getProperty("verifySSL");
+    if (verifySSL != null) {
+      builder.setVerifySSLCert(Boolean.valueOf(verifySSL));
+    }
```

Comment:
that's why we shouldn't define a default value within this code, right? So it always defaults to whatever ClientConfig defaults to.